packages/react-components/src/queries/utils/useTimeSeriesDataQuerySync.ts

```diff
@@ -2,18 +2,17 @@ import { useEffect } from 'react';
 import { StateCreator, create } from 'zustand';
 import { devtools } from 'zustand/middleware';
 
-export interface LatestAssetPropertyValueSync {
+export interface TimeSeriesDataSync {
```

Comment:
There are two separate data loaders / batches for the latest and historical time series requests. Shouldn't we have two separate sync states managed in the same store here?

Reply:
I considered that, but I thought that it is best that all time series data queries with the same refresh rate, refresh at the same time. I think that is what consumers would expect to happen. But you're right, since they are using different loaders, it doesn't actually improve performance in any way because they are different batch apis. I just wanted to ensure that data flows into charts at the same time.